lib/Features/FeaturesController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace MailPoet\Features;
+
+use MailPoet\Models\FeatureFlag;
+
+class FeaturesController {
+
+  // Define features below in the following form:
+  //   const FEATURE_NAME_OF_FEATURE = 'name-of-feature';
+  const FEATURE_DISPLAY_WOOCOMMERCE_REVENUES = 'display-woocommerce-revenues'; // may also have 'display_revenues' setting
+
+  // Define feature defaults in the array below in the following form:
+  //   self::FEATURE_NAME_OF_FEATURE => true,
+  public static $defaults = [
```

Comment:
I actually wanted `public const DEFAULTS = [...]` and I got confused thinking arrays in class constants are not supported in PHP 5.6. It turns out they are - https://3v4l.org/m1jkn (only not out of classes using "define"). The "public" is supported only since PHP 7.1 but I can easily drop that since it's the default. Now I'm only not sure about mocking this.

I'll have a look and improve it somehow. Either with const or as you said.

Reply:
Rewritten to a nicer version with private array.